frappe/handler.py

```diff
@@ -42,6 +42,11 @@ def handle():
 	return build_response("json")
 
 def execute_cmd(cmd, from_async=False):
+
+	from frappe.utils import sanitize_html
+
+	cmd = sanitize_html(cmd, strip=True)
+
```

Comment:
1. should be sanitized at client side?
2. why is this written above docstrings?

Reply:
I couldn't find anything to sanitize requests on the client-side so handled it here. However, added client-side filtering for form fields  of field type such as Data, Small Text, Text, Long Text.

Also, fixed the docstring thing (that was stupidity from my side).